main.c

```diff
@@ -559,10 +559,10 @@ static void cmdproc_thread_cleanup(void *arg)
 
 static void *tcmur_cmdproc_thread(void *arg)
 {
-        int ret;
 	struct tcmu_device *dev = arg;
 	struct tcmur_handler *rhandler = tcmu_get_runner_handler(dev);
 	struct pollfd pfd;
+        int ret;
```

Comment:
What happened here? I think you might have just cut and paste the original code and as a result took the spaces instead of tabs.